### DISCUSSION TITLE
Is AdMob available on mobile?

### DISCUSSION THREAD

**Discussion by LiberRuth** [unoplatform/uno#18096] [Q&A] [answered]
Is it available through Xamarin.Firebase.iOS.AdMob or Xamarin.GooglePlayServices.Ads?

**Comment by MartinZikmund** [comment, marked as answer]
@LiberRuth you can use any Android or iOS library as required in your application, as long as you reference the package specifically only on that target (e.g. `Xamarin.GooglePlayServices.Ads` only on Android). See [docs ](https://platform.uno/docs/articles/native-views.html) for more information.
